pages/common/transfersh.md

```diff
@@ -0,0 +1,28 @@
+# transfersh
+
+> An unofficial command-line client for transfer.sh.
+> More information: <https://github.com/AlpixTM/transfersh>.
+
+- Upload a file to transfer.sh:
+
+`transfersh {{path/to/file}}`
+
+- Upload a file showing a progress bar (requires Python package `requests_toolbelt`):
+
+`transfersh --progress {{path/to/file}}`
+
+- Upload a file using a different file name:
+
+`transfersh --name {{file_name}} {{path/to/file}}`
+
+- Upload a file to a custom transfer.sh server:
+
+`transfersh --servername {{upload.server.name}} {{path/to/file}}`
+
+- Upload all files in a directory recursively:
```

Comment:
from?
```suggestion
- Upload all files from a directory recursively:
```

Reply:
Not sure. `From` sounds like you're moving the files.

Reply:
Either would work fine here.